pandas/tseries/base.py

```diff
@@ -382,14 +393,18 @@ def argmin(self, axis=None):
             i8[mask] = np.iinfo('int64').max
         return i8.argmin()
 
-    def max(self, axis=None):
+    def max(self, axis=None, *args, **kwargs):
         """
-        return the maximum value of the Index
+        Return the maximum value of the Index or maximum along
```

Comment:
@jreback : Did you consider the reason that I added the comment was because nowhere in the documentation does it explain what the `axis` parameter is or does.  You get that clearly explaining parameters in documentation is good practice.